chain/client/src/stateless_validation/state_witness_producer.rs

```diff
@@ -34,6 +34,9 @@ impl Client {
         if !checked_feature!("stable", StatelessValidationV0, protocol_version) {
             return Ok(());
         }
+        let chunk_header = chunk.cloned_header();
+        let shard_id = chunk_header.shard_id();
```

Comment:
did you mean to use the shard_id as a the span field?

Reply:
yes. done.